### PR TITLE
Fix tone tracker rockets dealing high damage in frontier war

### PR DESCRIPTION
**Pull request by Gityeeter**
Tone's tracker rockets absolutely shreds harvester, I'm pretty sure this is not intended
https://user-images.githubusercontent.com/94848105/213841091-547660df-2536-4619-ae4d-ecaa7bd6ebe8.mp4

I have reduced the damage of the tracker rockets by sevenfold.

### CODE REVIEW

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut [hunk 1]
```diff
@@ -1749,7 +1749,7 @@ void function FW_createHarvester()
 	DispatchSpawn( trackerImc )
 	SetLocationTrackerRadius( trackerImc, 1 ) // whole map
 
-	// scores starts from 100, TeamScore means harvester health; TeamScore2 means shield bar
+	// scores starts from 100, TeamSc ore means harvester health; TeamScore2 means shield bar
```
Comment (ASpoonPlaysGames) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:1752]: accidental change?

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut [hunk 2]
```diff
@@ -1788,7 +1788,6 @@ void function OnHarvesterDamaged( entity harvester, var damageInfo )
 	int friendlyTeam = harvester.GetTeam()
 	int enemyTeam = GetOtherTeam( friendlyTeam )
 	int damageSourceID = DamageInfo_GetDamageSourceIdentifier( damageInfo )
-
```
Comment (ASpoonPlaysGames) [inline Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:1791]: accidental change?

File: Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut (hunk 1 above)
Comment (Gityeeter) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut:1752]: oops seems like it

Comment (Gityeeter) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut]: did i change it correctly, im not sure

Comment (ASpoonPlaysGames) [reply Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut]: indeed you did

**Comment by F1F7Y** [comment]
Would prefer to merge #579 first and then update this to use the new way of balancing, or just include this in #579

**Pull request closed without merging**